sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -91,118 +91,120 @@
    * @param <T>           The Type of the return, use byte[] to skip serialization.
    * @return A Mono Plan of type T.
    */
-  <T> Mono<T> invokeService(String appId, String method, Object request, HttpExtension httpExtension,
-                            Map<String, String> metadata, Class<T> clazz);
+  <T> Mono<T> invokeMethod(String appId, String methodName, Object request, HttpExtension httpExtension,
+                           Map<String, String> metadata, Class<T> clazz);
 
   /**
-   * Invoke a service method, using serialization.
+   * Invoke a service methodName, using serialization.
    *
    * @param appId         The Application ID where the service is.
-   * @param method        The actual Method to be call in the application.
+   * @param methodName        The actual Method to be call in the application.
    * @param request       The request to be sent to invoke the service, use byte[] to skip serialization.
    * @param httpExtension Additional fields that are needed if the receiving app is listening on
    *                      HTTP, {@link HttpExtension#NONE} otherwise.
    * @param type          The Type needed as return for the call.
    * @param <T>           The Type of the return, use byte[] to skip serialization.
    * @return A Mono Plan of type T.
    */
-  <T> Mono<T> invokeService(String appId, String method, Object request, HttpExtension httpExtension, TypeRef<T> type);
+  <T> Mono<T> invokeMethod(String appId, String methodName, Object request, HttpExtension httpExtension,
+                           TypeRef<T> type);
 
   /**
-   * Invoke a service method, using serialization.
+   * Invoke a service methodName, using serialization.
```

Comment:
```suggestion
   * Invoke a service method, using serialization.
```
And similar one for all the invokeMethod variants.